tests/benchmarks/constructs/CallCompiledClassCreationPosArgsConstant6.py

```diff
@@ -19,31 +19,37 @@
 #
 from __future__ import print_function
 
+import itertools
+
+
 class C(object):
-    def __init__(self, a,b,c,d,e,f):
+    def __init__(self, a, b, c, d, e, f):
         self.a = a
         self.b = b
         self.c = c
         self.d = d
         self.e = e
         self.f = f
 
+
 def calledRepeatedly():
     # force frame
     C
 
     # This is supposed to make a call to a compiled method, which is
     # being optimized separately.
-# construct_begin
+    # construct_begin
     C("some", "random", "values", "to", "check", "call")
     C("some", "other", "values", "to", "check", "call")
     C("some", "new", "values", "to", "check", "call")
 
-# construct_alternative
+    # construct_alternative
     pass
+
+
 # construct_end
```

Comment:
Here the auto-format misses the indentation. This stupid pre-processor of mine is something that ought to be replaced with Jinja2 one day soon. The support for Python in the Jinja2 mode is very excellent, and there not being a good way to edit them otherwise was part of the motivation to do it. Luckily this will map `if else endif` of Jinja2 pretty easily.

Reply:
I believe the change in indentation here is ok - the code looks for this anywhere on the line.